visual_behavior/data_access/utilities.py

```diff
@@ -1,4 +1,202 @@
+    """
+    ophys_sessions = cache.get_session_table()
+    if ophys_session_id not in ophys_sessions.index:
+        raise Exception('ophys_session_id not in session table')
+    experiments = ophys_sessions.loc[ophys_session_id].ophys_experiment_id
+    return experiments[0]
+
+
+def get_behavior_session_id_from_ophys_experiment_id(ophys_experiment_id, cache):
+    """finds the behavior_session_id associated with an ophys_experiment_id
+
+    Arguments:
+        ophys_experiment_id {int} -- 9 digit, unique identifier for an ophys_experimet
+        cache {object} -- cache from BehaviorProjectCache
+
+    Raises:
+        Exception: [description]
+
+    Returns:
+        int -- behavior_session_id, 9 digit, unique identifier for a behavior_session
+    """
+    ophys_experiments = cache.get_experiment_table()
+    if ophys_experiment_id not in ophys_experiments.index:
+        raise Exception('ophys_experiment_id not in experiment table')
+    return ophys_experiments.loc[ophys_experiment_id].behavior_session_id
+
+
+def get_ophys_session_id_from_ophys_experiment_id(ophys_experiment_id, cache):
+    """finds the ophys_session_id associated with an ophys_experiment_id
+
+    Arguments:
+        ophys_experiment_id {int} -- 9 digit, unique identifier for an ophys_experimet
+        cache {object} -- cache from BehaviorProjectCache
+
+    Raises:
+        Exception: [description]
+
+    Returns:
+        int -- ophys_session_id: 9 digit, unique identifier for an ophys_session
+    """
+    ophys_experiments = cache.get_experiment_table()
+    if ophys_experiment_id not in ophys_experiments.index:
+        raise Exception('ophys_experiment_id not in experiment table')
+    return ophys_experiments.loc[ophys_experiment_id].ophys_session_id
+
+
+def get_donor_id_from_specimen_id(specimen_id, cache):
+    """gets a donor_id associated with a specimen_id. Both donor_id
+        and specimen_id are identifiers for a mouse.
+
+    Arguments:
+        specimen_id {int} -- 9 digit unique identifier for a mouse
+        cache {object} -- cache from BehaviorProjectCache
+
+    Returns:
+        int -- donor id
+    """
+    ophys_sessions = cache.get_session_table()
+    behavior_sessions = cache.get_behavior_session_table()
+    ophys_session_id = ophys_sessions.query('specimen_id == @specimen_id').iloc[0].name  # noqa: F841
+    donor_id = behavior_sessions.query('ophys_session_id ==@ophys_session_id')['donor_id'].values[0]
+    return donor_id
+
+
+def model_outputs_available_for_behavior_session(behavior_session_id):
+    """
+    Check whether behavior model outputs are available in the default directory
+
+    :param behavior_session_id: 9-digit behavior session ID
+    :return: Boolean, True if outputs are available, False if not
+    """
+    model_output_dir = loading.get_behavior_model_outputs_dir()
+    model_output_file = [file for file in os.listdir(model_output_dir) if str(behavior_session_id) in file]
+    if len(model_output_file) > 0:
```

Comment:
The logic block starting at 199 should be rewritten to be one statement: `return len(model_output_file) > 0`

(one of my pet-peeves)

Reply:
fixed. thanks for the suggestion.